src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.body.querySelector('ul');
+const listOfEmployeesChildrens = [...listOfEmployees.children];
+
+listOfEmployees.prepend(...sortList(listOfEmployeesChildrens));
+getEmployees(listOfEmployeesChildrens);
+
+function sortList(array) {
+  const parser = (string) => {
+    return parseInt(string.replaceAll('$', '').replaceAll(',', ''));
```

Comment:
The `replaceAll` method is used here to remove '$' and ',' from the salary string. Note that `replaceAll` is not supported in Internet Explorer. If compatibility with older browsers is a concern, consider using `replace` with a regular expression.